modules/mod-mp3/ExportMP3.cpp

```diff
@@ -1324,15 +1241,11 @@ int MP3Exporter::InitializeStream(unsigned channels, int sampleRate)
    // Set the channel mode
    MPEG_mode mode;
 
-   if (channels == 1 || mChannel == CHANNEL_MONO) {
+   if (channels == 1)
       mode = MONO;
-   }
-   else if (mChannel == CHANNEL_JOINT) {
+   else
       mode = JOINT_STEREO;
```

Comment:
But is it really a preferred option? I think joint stereo reduces the quality.

Reply:
I'm fairly confident that it doesn't.  Joint stereo does a per-frame M/S encoding, rather than L/R encoding where the channels are similar, and where they aren't, it switches back to L/R encoding. LAME makes this decision for every MPEG frame. 

LAME also has options to force M/S encoding (`-m f` instead of `-m j`), which I hope we don't use.

Reply:
Yeah, judging by https://github.com/enzo1982/freac/discussions/329 and LAME source code you are correct

Reply:
> Yeah, judging by [enzo1982/freac#329](https://github.com/enzo1982/freac/discussions/329) and LAME source code you are correct

And here: https://wiki.hydrogenaud.io/index.php?title=Joint_stereo#MP3